### DISCUSSION TITLE
can´t update via installer - ERROR

### DISCUSSION THREAD

**Discussion by andrejetcat** [flybywiresim/aircraft#6500] [Help] [answered]
Hello together, I want to update the FBM via the Installer today. But It seems that it stucks while the livery module.

also I can´t switch over to DEV Version.

it there a problem on FBW´w side, or can anybody help me out :( 
![installer](https://user-images.githubusercontent.com/72918017/148559841-af6b7e6c-b7c7-4716-8fcb-bffb49d4bea1.jpg)

**Comment by kdtheisen** [comment]
me too!

**Comment by Daboj1337** [comment, marked as answer]
Just delete FBW from the Community folder and install it again. I had to do this too because of this error.
